easybuild/easyconfigs/x/xcb-proto/xcb-proto-1.11-intel-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'xcb-proto'
+version = '1.11'
+
+homepage = 'http://xcb.freedesktop.org/'
+description = """The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint,
+latency hiding, direct access to the protocol, improved threading support, and extensibility."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = ['http://xcb.freedesktop.org/dist/']
+sources = [SOURCELOWER_TAR_GZ]
+
+python = 'Python'
+pyver = '2.7.11'
+versionsuffix = '-%s-%s' % (python, pyver)
+builddependencies = [(python, pyver)]
```

Comment:
@rjeschemi: the current `libX11` depends on `libxcb`, and `libxcb` depends on `xcb-proto`. `xcb-proto` is a bunch of Python script, so it needs Python to use. I am heavily suspect that `xcb-proto` is only needed to build `libxcb`, so at the end `libxcb` must be `Python` free.